opus/application/static_media/js/browse.js

```diff
@@ -2239,4 +2224,23 @@ var o_browse = {
         $("#browse .gallery").empty();
         $("#browse .op-data-table tbody").empty();
     },
+
+    downloadCSV: function(obj) {
+        let colStr = opus.prefs.cols.join(',');
+        let selectionsHash = [];
+        for (let param in opus.selections) {
```

Comment:
This will utterly fail with the new multiple-input selections feature. In fact if you have multiple input rows and then select download CSV you'll get a download error. Sadly, this appears to be in the production version too! So it needs to be fixed.

Look at the new function o_hash.GetHashStrFromSelections and that should be able to replace all this code.